bc/subscription/api_views.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from pprint import pformat
+
+from django.http import HttpRequest, HttpResponse
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from .api_serializers import WebhookSerializer
+
+logger = logging.getLogger(__name__)
+
+
+@api_view(["POST"])
+def handle_cl_webhook(request: HttpRequest) -> HttpResponse:
```

Comment:
OK, but, like, does this have a url somewhere?

Reply:
It does not. I'll create an url for this view.